src/Microsoft.AspNetCore.SystemWebAdapters/HttpCachePolicyBase.cs

```diff
@@ -12,17 +12,14 @@ public class HttpCachePolicyBase
 
     public virtual void SetCacheability(HttpCacheability cacheability) => throw new NotImplementedException();
 
-    public virtual void SetCacheability(HttpCacheability cacheability, string field) => throw new NotImplementedException();
-    
-    [SuppressMessage("Microsoft.Naming", "CA1716:IdentifiersShouldNotMatchKeywords", MessageId = "Date",
-        Justification = "Matches HttpCachePolicy class")]
+    [SuppressMessage("Naming", "CA1716:Identifiers should not match keywords", Justification = Constants.ApiFromAspNet)]
```

Comment:
Looks like you've removed the SetCacheability overload here... Was that intentional?

Reply:
Yes - it wasn't implemented on HttpCachePolicy itself so we couldn't forward it in the wrapper correctly. Happy to take a fix that implements it on all three as a follow up :) I've added tests to validate that the three (i.e. HttpContext/HttpContextBase/HttpContextWrapper) are all kept in sync when adding APIs and this case was causing failure